docs/problems/NoResolution.md

```diff
@@ -15,7 +15,7 @@ This problem indicates that TypeScript can’t find any file with a supported fi
 
```

Comment:
☝️ [There's another one](https://github.com/arethetypeswrong/arethetypeswrong.github.io/pull/174/files#diff-c2d00d4f3e471929d7d29bdf358d6af82ab425c7344c8a099c46909f1fb9ae84R9) -- "Consequnces" -- would you mind including that? I was just about to open a PR myself, but since you're already on it 😎

Reply:
good catch. thanks